app/views/layouts/application.html.erb

```diff
@@ -4,7 +4,7 @@
     <title>Boardemy</title>
     <%= csrf_meta_tags %>
     <%= csp_meta_tag %>
-
+    <script src="https://kit.fontawesome.com/b0e17e7f5a.js" crossorigin="anonymous"></script>
```

Comment:
I think it is better to install the `fontawesome` js library if you want to use it.